jetty-util/src/main/java/org/eclipse/jetty/util/RolloverFileOutputStream.java

```diff
@@ -229,13 +229,32 @@ void setFile(ZonedDateTime now)
         File backupFile = null;
         try (AutoLock l = _lock.lock())
         {
+            if (_filename == null)
+            {
+                throw new IOException("Log directory name is null");
+            }
+            _filename = _filename.trim();
+            if (_filename.isEmpty())
+            {
+                throw new IOException("Log directory name is empty");
+            }
```

Comment:
```suggestion
            if (StringUtil.isBlank(_filename)
                throw new IOException("Log directory name is null or empty");
            _filename = _filename.trim();
```

Reply:
Hi Greg,
Thanks for the comments.
I'll simplify fix:
- remove check for null because we have it in the class constructor;
- remove trimming of filename because space is allowed at a begin of filename (I don't like it, but ...);
- remove check for empty filename.
I'll will commit simplified fix to jetty-10 and create new pull request.
I'll close current pull request if I have rights for it.